tinytodo-go/internal/app/server/context_test.go

```diff
@@ -0,0 +1,377 @@
+package server
+
+import (
+	"code.byted.org/binaryauthorization/tinytodo-go/internal/app/server/entitystore"
+	"code.byted.org/binaryauthorization/tinytodo-go/internal/app/server/entitystore/action"
+	"code.byted.org/binaryauthorization/tinytodo-go/internal/app/server/entitystore/entitytype"
+	"encoding/json"
+	"github.com/cedar-policy/cedar-go"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"os"
+	"path"
+	"testing"
+)
+
+func readFile(t *testing.T, prefix, filename string) []byte {
```

Comment:
Hmmm, I'm not really understanding the need for prefix and filename in this helper. It seems like you could just pass a single argument.

Reply:
Will change this